File: Programming Basics with Python/02 Conditional statements/More exercises/08_Fuel_Tank_Part_2.py
fuel = input().lower()
fuel_quantity = float(input())
card = input().lower()
diesel_price = 2.33
gasoline_price = 2.22
gas_price = 0.93
result = 0
if card == 'yes':
    diesel_price -= 0.12
    gasoline_price -= 0.18
    gas_price -= 0.08
if 20 <= fuel_quantity <= 25:
    discount = 0.92
elif fuel_quantity > 25:
    discount = 0.9
else:
    discount = 1
if fuel == 'diesel':
    result = diesel_price * discount * fuel_quantity
elif fuel == 'gasoline':
    result = gasoline_price * discount * fuel_quantity
elif fuel == 'gas':
    result = gas_price * discount * fuel_quantity
print(f'{result:.2f} lv.')

File: JS Basics/02 Conditional Statements - More Exercises/08. Fuel Tank - Part 2.js
function tank(fuel, liters, card) {
    liters = Number(liters)
    let price = 0
    if (fuel == "Gasoline") {
        price = 2.22;
        if (card == "Yes") {
            price = 2.22 - 0.18;
        }
    }
    else if (fuel == "Gas") {
        price = 0.93;
        if (card == "Yes") {
            price = 0.93 - 0.08;
        }
    }
    else if (fuel == "Diesel") {
        price = 2.33
        if (card == "Yes") {
            price = 2.33 - 0.12;
        }
    }
    price *= liters;
    if (20 <= liters & liters <= 25) {
        price *= 0.92;
    }
    else if (liters > 25) {
        price *= 0.9
    }
    console.log(`${price.toFixed(2)} lv.`)
}
tank("Diesel", 19, "No")
